biodatasets/sciq/sciq.py

```diff
@@ -0,0 +1,154 @@
+            schema="source",
+            subset_id=f"sciq",
+        ),
+        # BigBio schema: question answering
+        BigBioConfig(
+            name=f"sciq_bigbio_qa",
+            version=BIGBIO_VERSION,
+            description="SciQ simplified BigBio schema",
+            schema="bigbio_qa",
+            subset_id="sciq",
+        ),
+    ]
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "question": datasets.Value("string"),
+                    "distractor1": datasets.Value("string"),
+                    "distractor2": datasets.Value("string"),
+                    "distractor3": datasets.Value("string"),
+                    "correct_answer": datasets.Value("string"),
+                    "support": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "bigbio_qa":
+            features = schemas.qa_features
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        dl_dir = dl_manager.download_and_extract(_URLs)
+        data_dir = os.path.join(dl_dir, "SciQ dataset-2 3")
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "train.json"),
+                    "split": datasets.Split.TRAIN,
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "valid.json"),
+                    "split": datasets.Split.VALIDATION,
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "test.json"),
+                    "split": datasets.Split.TEST,
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        with open(filepath, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        if self.config.schema == "source":
+            for i, d in enumerate(data):
+                yield i, d
+        elif self.config.schema == "bigbio_qa":
+            document_id = split
+            for i, d in enumerate(data):
+                id = f"{split}_{i}"
+                yield id, {
+                    "id": id,
+                    "question_id": id,
+                    "document_id": document_id,
+                    "question": d["question"],
+                    "type": "",  # TODO - not provided in original data
+                    "context": d["support"],
```

Comment:
```suggestion
                    "context": d["support"],
                    "choices": "<List[str] of choices>"
```
@j-chim can you fill in the choices here from the datase`d`?

Reply:
thanks @galtay! both should be done in the latest commit